saleor/settings.py

```diff
@@ -534,6 +539,10 @@ def get_host():
         "task": "saleor.giftcard.tasks.deactivate_expired_cards_task",
         "schedule": crontab(hour=0, minute=0),
     },
+    "delete-old-export-files": {
+        "task": "saleor.csv.tasks.delete_old_export_files",
+        "schedule": crontab(hour=1, minute=0),
```

Comment:
wouldn't once a day be enough?

Reply:
It is once per day, at a specified time.